core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -1059,10 +1059,14 @@ public Health getHealth() {
 
    @Override
    public ClusterExecutor executor() {
-      if (globalComponentRegistry.getStatus() != ComponentStatus.RUNNING) {
+      authzHelper.checkPermission(AuthorizationPermission.ADMIN);
```

Comment:
Why not the `EXEC` permission?

Reply:
Whoops, yeah it should be `EXEC` :)

Reply:
Actually thinking about this more I can see this as `ADMIN` like I had before as well since it is at the cache manager level and many users may want to start a cache etc. I think having it more strict for now is a bit safer.